admin/src/main/scala/io/buoyant/admin/Admin.scala

```diff
@@ -117,7 +117,7 @@ object Admin {
   }
 }
 
-class Admin(val address: InetSocketAddress, tlsCfg: Option[TlsServerConfig], workers: Int, stats: StatsReceiver) {
+class Admin(val address: InetSocketAddress, tlsCfg: Option[TlsServerConfig], workers: Int, stats: StatsReceiver, securityConfig: Option[AdminSecurityConfig]) {
```

Comment:
nit: you probably could format the class definition for readability to something like:

```scala
class Admin(
  val address: InetSocketAddress,
  tlsCfg: Option[TlsServerConfig],
  workers: Int, 
  stats: StatsReceiver, 
  securityConfig: Option[AdminSecurityConfig]) {
...
```

Reply:
Updated the PR according to your suggestion.